tests/simplex/data_out_write_vtk_01.cc

```diff
@@ -103,24 +111,38 @@ main()
 {
   initlog();
 
-  {
-    const unsigned int dim = 2;
-    test<dim>(Simplex::FE_P<dim>(2) /*=degree*/, 1);
-    test<dim>(FESystem<dim>(Simplex::FE_P<dim>(2 /*=degree*/), dim), dim);
-    test<dim>(FESystem<dim>(Simplex::FE_P<dim>(2 /*=degree*/),
-                            dim,
-                            Simplex::FE_P<dim>(1 /*=degree*/),
-                            1),
-              dim + 1);
-  }
-  {
-    const unsigned int dim = 3;
```

Comment:
If I see this right, then you are dropping the 3d part of the test. Is this on purpose? That would be unfortunate because we then no longer test this functionality...

Reply:
No, I am not. Diff is only confused by the new indentation.